src/components/common/PlainDraggable.vue

```diff
@@ -90,6 +90,7 @@ export default {
   },
 
   destroyed() {
+    this.$emit('destroyed')
```

Comment:
Can you add `{ draggable }` as payload to this event?

Reply:
Sure, I will @bzabos!  It was not required in this context so I hadn't added.